tools/pythonpkg/tests/fast/test_expression.py

```diff
@@ -834,3 +834,14 @@ def test_sort(self):
         rel2 = rel.sort(b.desc().nulls_last())
         res = rel2.b.fetchall()
         assert res == [('c',), ('b',), ('a',), ('a',), (None,)]
+
+    def test_aggregate(self):
+        con = duckdb.connect()
+        con.execute("PRAGMA threads=4")
+        con.execute("PRAGMA verify_parallelism")
+
+        rel = con.sql("select * from range(1000000) t(a)")
+        # Also test multiple reads
+        count = FunctionExpression("count", "a").cast("int")
+        assert rel.aggregate([count]).execute().fetchone()[0] == 1000000
```

Comment:
Can we add a test where we pass a wrong type to the aggregate function, e.g. `rel.aggregate(42)`, `rel.aggregate([42])`?